Jenkinsfile

```diff
@@ -1186,7 +1201,20 @@ pipeline {
                             run_if_pr: true,
                             run_if_landing: false,
                             job_status: job_status_internal
-                        )
+                        ),
+                        'Functional Hardware Large MD on SSD': getFunctionalTestStage(
+                            name: 'Functional Hardware Large MD on SSD',
+                            pragma_suffix: '-hw-large-md-on-ssd',
+                            label: params.FUNCTIONAL_HARDWARE_MEDIUM_MD_ON_SSD_LABEL,
```

Comment:
I think you want a large label here yes?  I believe this PR took a Medium cluster offline sometime Saturday due to this.  I have brought it back online but definitely need to fix this before pushing this PR any more.

Reply:
Yes, thanks.  I fixed it in https://github.com/daos-stack/daos/commit/d3b7591b35130243add3125fa7712b6c8931996d and picked up the change in a merge with master.